README.md

```diff
@@ -63,8 +63,11 @@ export PATH=${PATH}:/Applications/Postgres.app/Contents/Versions/11/bin/
 
 To switch redis on you'll need to install it locally. On a OSX we've used brew for this. To use redis caching you need to switch it on by changing the config for development:
 
-        REDIS_ENABLED = True
+```
+REDIS_ENABLED = 1
+```
 
+Note that if you're running Celery with Docker locally, then you'll also need to set `REDIS_URL=redis://host.docker.internal:6379`.
```

Comment:
Optional: Can we set this environment variable in the dockerfile as a default to avoid people having to do this manually themselves so that it 'just works'?

Like at the bottom of https://github.com/alphagov/notifications-api/pull/3450/commits/d567d1529db7778b73205f73f9b5ec1a94b66461

Reply:
Good point - I missed that in your one.

Amended in https://github.com/alphagov/notifications-api/pull/3467/commits/6092baea643e5058237fd0d48e06f9d8f5918a65.